ledger/test-common/BUILD.bazel

```diff
@@ -15,10 +15,24 @@ load(
 )
 load(
     "//daml-lf/language:daml-lf.bzl",
+    "lf_dev_version",
+    "lf_latest_version",
+    "lf_preview_version",
     "lf_stable_version",
 )
 load("@scala_version//:index.bzl", "scala_major_version")
 
+# Tuples of (lf_version, name_suffix)
+# The DAML files are compiled once for each daml-lf version listed here.
+test_common_configurations = [
+    (lf_stable_version, ""),
+    (lf_latest_version, "-latest"),
+    (lf_dev_version, "-dev"),
+] + [(
+    lf_preview_version[0],
+    "-preview",
```

Comment:
`lf_preview_version` contains at most one version, but may be empty. 
Think about it as an option (I did not find option in starlark, so I used list)

you can look how I deal with that in [sandbox-classic-tests](https://github.com/digital-asset/daml/blob/f13c96bddef5a71b4de876535462fc1f9a10c6d4/ledger/sandbox-classic/BUILD.bazel#L242) (It is ugly I know)

Reply:
That's what happens here as well, see the line below:
```
if lf_preview_version != [] else []
```

Reply:
I missed that. Sorry for the noise.

Reply:
I have updated the code to use the same list comprehension as in sandbox-classic-tests.